wilkes/roster.json

```diff
@@ -132,7 +132,9 @@
       "github": "ArvinJhurry",
       "materials": [
         {
-          "type":"cv"
+          "type":"video",
+          "url":"<https://youtu.be/FkmDhmAEVv8>"
+
         }
```

Comment:
@ArvinJhurry 
2 things
1. you need to keep the cv "object"
2. We only need to see the endpoint of the youtube video and not the full url, can you please change to "url": "FkmDhmAEVv8"

Reply:
Hi, I have made some changes, is this what you meant?